src/chrome/content/rules/IPython.org.xml

```diff
@@ -5,13 +5,18 @@
 		- nbviewer ²
 
 	¹: Timeout
-	²: Hostname not in cert
+	²: HTTP redirects but HTTPS does not
 -->
 <ruleset name="IPython.org">
 
 	<target host="ipython.org" />
 	<target host="www.ipython.org" />
 	
+	<target host="nbviewer.ipython.org" />
+
+	<rule from="^http://nbviewer\.ipython\.org/"
+		to="https://nbviewer.jupyter.org/" />
+
 	<rule from="^http://(?:www\.)?ipython\.org/"
```

Comment:
They should, but if we want to get rid of them, we should script that rather than enforce it on a ruleset-by-ruleset basis.